### PR TITLE
Rst proofing

### PR DESCRIPTION
**Pull request by dpryan79**

### CODE REVIEW

File: deeptools/bamCompare.py [hunk 1]
```diff
@@ -127,13 +127,13 @@ def getOptionalArgs():
...
     optional.add_argument('--ratio',
-                          help='The default is to output the log2ratio between the '
+                          help='The default is to output the log2ratio of the '
                           'two samples. The reciprocal ratio returns the '
                           'the negative of the inverse of the ratio '
                           'if the ratio is less than 0. The resulting '
                           'values are interpreted as negative fold changes. '
-                          '*NOTE*: Only when --ratio subtract, the options --normalizeTo1x or '
-                          '--normalizeUsingRPKM can be used.',
+                          '*NOTE*: Only with --ratio subtract can --normalizeTo1x or '
```
Comment (bgruening) [inline deeptools/bamCompare.py]: This sounds strange. `--normalizeTo1x or ... can only be used with "--ratio subtract"`

File: deeptools/plotPCA.py [hunk 2]
```diff
@@ -40,9 +40,9 @@ def plotCorrelationArgs():
                           required=True)
 
     required.add_argument('--plotFile', '-o',
-                          help='File name to save the plot. '
-                          'The given file ending will be used '
-                          'to determine the image format, for example: '
+                          help='File name to save the plot to. '
+                          'The extension determines the file format. '
```
Comment (bgruening) [inline deeptools/plotPCA.py]: If not overwritten by `--plotFileFormat`

File: docs/content/example_gallery.rst [hunk 3]
```diff
@@ -19,15 +19,15 @@ The following image demonstrates that enhancer regions are typically
...
+yellow and blue tiles indicate a large numbers of reads that were
+sequenced (indicative of open chromatin) and black spots indicate
 missing data points. An appropriate labeling of the y-axis was
 neglected.
 
 **Fast Facts:**
 
 * `computeMatrix` mode: reference-point
-* *regions file*: BED file with typical enhancer regions from `Whyte et al., 2013 <http://dx.doi.org/10.1016/j.cell.2013.03.035>`__ (download `here <https://raw.github.com/fidelram/deepTools/master/gallery/Whyte_TypicalEnhancers_ESC.bed>`__)
+* *regions file*: BED file with typical enhancer regions from `Whyte et al., 2013 <http://dx.doi.org/10.1016/j.cell.2013.03.035>`__ (download `here <https://raw.github.com/fidelram/deepTools/release-1.6/gallery/Whyte_TypicalEnhancers_ESC.bed>`__)
```
Comment (bgruening) [inline docs/content/example_gallery.rst]: This reminds me ... when do we plan to do the merge? This should be changed back at time of merging.

File: docs/content/example_gallery.rst [hunk 4]
```diff
@@ -104,14 +104,12 @@ Visualizing the GC content for mouse and fly genes
 
 It is well known that different species have different genome GC
 contents. Here, we used two bigWig files where the GC content was
-calculated for 50 bp windows along the genome of mice and flies and
-visualized the scores for gene regions. You can find the bigWig files in
-our `Galaxy's data
-library <http://deeptools.ie-freiburg.mpg.de/library>`__.
+calculated for 50 base windows along the genome of mice and flies and
```
Comment (bgruening) [inline docs/content/example_gallery.rst]: Time to split the repos into an `deeptools gallery` one and `deeptools test data`?

File: docs/content/example_gallery.rst (hunk 3 above)
Comment (dpryan79) [inline docs/content/example_gallery.rst]: I suspect once we're all back in (so ca. Jan. 12).

File: deeptools/bamCompare.py (hunk 1 above)
Comment (dpryan79) [inline deeptools/bamCompare.py]: Yeah, there's still a lot of weird English, though its at least more tolerable now.

**Pull request merged**